include/rocksdb/options.h

```diff
@@ -1144,6 +1144,22 @@ struct DBOptions {
   // not be used for recovery if best_efforts_recovery is true.
   // Default: false
   bool best_efforts_recovery = false;
+
+  // It defines How many times call db resume when retryable BGError happens.
```

Comment:
Does somebody want to tune these knobs? How about infinite recovery attempts with a one second gap?

Reply:
Yes, that's a good idea. We probably just need an option to enable/disable the functionality in case of bugs.

Reply:
@anand1976 You mean, I can remove the options of max_bgerror_resume_count and bgerror_resume_retry_interval, set value directly in the resume function. Instead, we give a option for user to enable or disable auto resume? The resume will be infinite attempts as suggested by Andrew?

Reply:
@zhichao-cao Thinking about this some more, I think there's value in allowing the retry interval to be adjusted. The flushes initiated by the recovery threads share the same background thread pool as regular flushes, so if you have a mix of good and bad DBs, changing the retry interval will allow a user to control resource consumption for recovery. The retry count is debatable. We could set the default to infinite and user can set it to 0 to disable. I don't know if there would be a reason to choose some value in between. Again, maybe to limit resource consumption.